src/ContentHubClient.php

```diff
@@ -676,18 +676,22 @@ public function deleteWebhook($uuid)
     /**
      * Updates a webhook from the active subscription.
      *
-     * @param $uuid
-     *   The UUID of the webhook to update
+     * @param string $uuid
+     *   The UUID of the webhook to update.
+     * @param array $options
+     *   What to change in the webhook: url, version, disable_retries, etc.
      *
      * @return \Psr\Http\Message\ResponseInterface
      *
      * @throws \GuzzleHttp\Exception\RequestException
      */
-    public function updateWebhook($uuid, $url)
+    public function updateWebhook($uuid, array $options)
```

Comment:
Let's get a list of "acceptable" values that could be in $options so that we don't just put anything. Loop over the acceptable value keys and only use their values.